dev/io.openliberty.accesslists.internal/src/io/openliberty/accesslists/filterlist/FilterListFastStr.java

```diff
@@ -390,4 +388,23 @@ public void addEntry(int hashcode, int length) {
 
     }
 
+    /**
+     * Create a FilterListFastStr if possible, if not fall back to FilterListStr
+     * 
+     * @param elements
+     * @return the non null, but could be empty, FilterListStr
+     */
+    public static FilterListStr create(String[] elements) {
+        FilterListStr list;
+        list = new FilterListFastStr();
+        if (elements != null) {
+            if (list.buildData(elements) == false) {
+                list = new FilterListSlowStr();
+                list.buildData(elements);
+            }
+            list.setActive(true);
+           
+        }
+        return list;
+    }
```

Comment:
Do we want to update the copyright here since this create method was added?

Reply:
Done